libraries/autoplayDetection/autoplay.js

```diff
@@ -32,12 +32,19 @@ function startDetection() {
     .play()
     .then(() => {
       autoplayEnabled = true;
-      videoElement.pause();
+      // if the video is played on a WebView with playsinline = false, this stops the video, to prevent it from being displayed fullscreen
+      videoElement.src = '';
     })
     .catch(() => {
       autoplayEnabled = false;
     });
 }
 
+const version = navigator.userAgent.match(/iPhone OS (\d+)_(\d+)/)
+if (version !== null && parseInt(version[1]) < 17 && !navigator.userAgent.includes('Safari')) {
+  // skip autodetection on iOS 16 WebView
+  return
```

Comment:
This return is not in a function and fails the linter

Reply:
Oops my bad, thanks for pointing this out! I'm used to TypeScript + Webpack taking care of this automatically for me, and did not take the time to test it locally on Prebid 😅 I'll fix this tomorrow.

Reply:
I have fixed the code and rebased on master, it's all good now @patmmccann 👍